requirements.txt

```diff
@@ -12,6 +12,7 @@ nose==1.3.7
 
 # to generate documentation
 Sphinx==3.2.1
+docutils==0.16   # pinning this because it breaks at 0.18
```

Comment:
Docutils 0.18 is causing problems.  Sphinx 3.2.1 only needs docutils >= 0.11, and 0.16 is working fine.  So I just pinned it there and everything resolves and builds fine.